src/Checkbox.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import cx from 'classnames';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+
+/**
+ * @module Checkbox
+ */
+class Checkbox extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			checked: props.checked || false
+		};
+		this.onChange = this.onChange.bind(this);
+	}
+
+	onChange(e) {
+		this.setState({ checked: e.target.checked });
+	}
+
+	render() {
+		const {
+			checked, // eslint-disable-line no-unused-vars
+			children,
+			className,
+			id,
+			label,
+			name,
+			value,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			'minTouchHeight',
+			className
+		);
+
+		const elId = id || `${name}-${value}`;
+
+		return (
+			<Flex align='center'
+				className={classNames}
+				{...other}>
+				<FlexItem shrink>
+					<input	type='checkbox'
+						name={name}
```

Comment:
My information may be out of date, but a good practice used to be making `name` and `id` attributes have the same value